components/team/MemberCard.tsx

```diff
@@ -0,0 +1,18 @@
+import { Card, CardContent } from '@mui/material'
+
+function MemberCard({ member }) {
```

Comment:
Let's be consistent and name everything `TeamMember`, so, this file and component should be named `TeamMemberCard`. The same thing should be applied to all the code you wrote.